prometheus/filter.go

```diff
@@ -113,6 +114,14 @@ type labelPairs struct {
 	Label []*dto.LabelPair `protobuf:"bytes,1,rep,name=label" json:"label,omitempty"`
 }
 
-func (l *labelPairs) Reset()         {}
-func (l *labelPairs) String() string { return proto.CompactTextString(l) }
-func (*labelPairs) ProtoMessage()    {}
+func (l *labelPairs) Reset() {}
+
+func (l *labelPairs) String() string {
+	var a []string
+	for _, lbl := range l.Label {
+		a = append(a, lbl.String())
+	}
+	return strings.Join(a, "\n")
+}
```

Comment:
This change is identical to one made in idpe, since these two files line up exactly between the two projects